src/services/stepsService.ts

```diff
@@ -663,6 +664,7 @@ export class StepsService {
         type,
         defaultValue,
         description,
+        ...(type === "number" || "integer") && {step : 1},
```

Comment:
Hey @tplevko there is one minor issue with deepscan that points this out, https://deepscan.io/dashboard/#view=project&tid=19033&pid=22364&bid=658659&subview=pull-request&prid=1623867

So, best thing here is probably to write it like `...(type === "number" || type === "integer") && {step : 1},` instead of `...(type === "number" || "integer") && {step : 1},`.

Reply:
Thanks for pointing that out - updated